robot/robot.py

```diff
@@ -8,12 +8,14 @@
 from robot.motor import MotorBoard
 from robot.power import PowerBoard
 from robot.servo import ServoBoard
+from robot import __VERSION__
 
 _PathLike = Union[str, Path]
 
 LOGGER = logging.getLogger(__name__)
 
 
+
```

Comment:
This additional new line shouldnt be there. There should be 2 blank lines between top-level constructs

Reply:
oops, fixed in c5123303adc7cd9f4d947820a623450ca01cf0e3